### ISSUE TITLE
Creating a new venv using slap-1.13.0 results in virtual-env with missing `activate`

### ISSUE THREAD

**Issue by ndejong** [NiklasRosenstein/slap#135]
Latest slap-1.13.0 creates  venv that is missing `.local/venvs/<path>/bin/activate`  

Workaround for the moment has been to rollback to 1.12.0 

For example:-
```shell
❯ pip install slap-cli==1.13.0

❯ slap venv -cg foobar
creating global environment "foobar" (using python3)

❯ slap venv -ag foobar
activating global environment "foobar"
bash: /home/user/.local/venvs/foobar/bin/activate: No such file or directory

❯ ls -al ~/.local/venvs/foobar/bin/
total 36K
drwxrwxr-x 2 user user   8 2024-02-23 13:15 .
drwxrwxr-x 5 user user   8 2024-02-23 13:15 ..
-rwxrwxr-x 1 user user 250 2024-02-23 13:15 pip
-rwxrwxr-x 1 user user 250 2024-02-23 13:15 pip3
-rwxrwxr-x 1 user user 250 2024-02-23 13:15 pip3.10
lrwxrwxrwx 1 user user   7 2024-02-23 13:15 python -> python3
lrwxrwxrwx 1 user user  16 2024-02-23 13:15 python3 -> /usr/bin/python3
lrwxrwxrwx 1 user user   7 2024-02-23 13:15 python3.10 -> python3

```

**Comment by NiklasRosenstein** [comment]
Hi @ndejong, thanks for reporting this issue. It seems I made a mistake and made it pass `--ugprade` instead of `--upgrade-deps` to `python3 -m venv`!

**Comment by NiklasRosenstein** [comment]
Fixed in 4ec5e96e and 1.13.1!
